apps/nextjs-website/react-components/types/Form/Form.types.ts

```diff
@@ -3,17 +3,38 @@ import { CtaButtonProps, type CommonProps } from '../common/Common.types';
 export interface FormProps extends CommonProps {
   readonly title: string;
   readonly subtitle: string;
-  readonly privacyText: string | JSX.Element;
-  readonly privacyLink: string;
-  readonly privacyLinkText: string;
+  readonly privacyLinkRecaptchaPolicy: string;
+  readonly privacyLinkTextRecaptchaPolicy: string;
+  readonly privacyLinkRecaptchaTerms: string;
+  readonly privacyLinkTextRecaptchaTerms: string;
   readonly theme: 'light' | 'dark';
   readonly backgroundImage?: string;
   readonly ctaButtons: ReadonlyArray<CtaButtonProps>;
+  readonly showFirstName?: boolean;
+  readonly showLastName?: boolean;
+  readonly showEmail?: boolean;
```

Comment:
I see in `FormData` below that `email` is required.

What happens if `showEmail` is `false`? Are we just passing an empty string?

I would either:
1. Remove `showEmail` and leave `email` as required
2. Make `email` optional like `firstName` and `lastName` (although it is weird for both of them to be optional, but that's not a code-related issue).